docs/decisions/README.md

```diff
@@ -0,0 +1,4 @@
+# Architectural design records
+The idea is to provide a basic integrated ADR generation for #{your project}
```

Comment:
I think we can replace #{your project} with Factiva API Clint

Reply:
nice catch!